config/settings.yml

```diff
@@ -32,13 +32,13 @@ binaries:
   pdftk: pdftk
   clamdscan: /usr/bin/clamdscan
 
-db_encryption_key: f01ff8ebd1a2b053ad697ae1f0d86adb48ebb708021e4c76c3807d37f6b4e389d5aa45ea171f2f5074222784c1ee2bb8272390d1b9517a7a6987c22733ef00b2
+db_encryption_key: f01ff8ebd1a2b053ad697ae1f0d86adb
```

Comment:
In Ruby 2.3, OpenSSL would just truncate this key to 32 bytes.
In Ruby 2.4, OpenSSL no longer truncates, but now raises an error because it expects a 32 byte key.
[Reference](https://github.com/ruby/openssl/issues/116#issuecomment-286115710)

Reply:
Assuming this change is backward compatible with instances running Ruby 2.3 (which I think it is?), might be wise to PR & merge on it's own, preemptively to this PR upgrading ruby, to reduce risk.

This will require updates to `credstash` as well, which again, is probably wise to do in advance of the ruby upgrade if we can.

Reply:
Yes, it is backward compatible with Ruby 2.3. 
* All my specs pass with the truncated keys
* I did manually:
  * Create a db object that uses the `db_encryption_key` in a Ruby 2.3/long key environment.
  * Then I switched branches to the Ruby 2.4/short key environment and verified that I could retrieve the previously saved data.

These specific changes are only used in local dev/testing, with the other environments values being held in `credstash`, as you mentioned.

I'll open a separate PR for these truncations, and perhaps a devops engineer can truncate the values for `dev` and/or `staging` for now for additional manual testing.